src/main/java/com/exactpro/th2/common/schema/event/EventBatchSubscriber.java

```diff
@@ -62,10 +75,15 @@ protected int extractCountFrom(EventBatch message) {
     }
 
     @Override
-    protected String toShortDebugString(EventBatch value) {
+    protected String toShortTraceString(EventBatch value) {
         return toJson(value);
     }
 
+    @Override
+    protected String toShortDebugString(EventBatch value) {
+        return String.format("EventBatch: parent_event_id = %s", value.getParentEventId().getId());
```

Comment:
Please avoid the String.format method for simple cases, because it adds additional overhead for performance.